hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -107,28 +107,44 @@
     {
       name: pattgen_op_cg
       desc: '''
-            The following coverage strategy is implemented:
-            * The counters are covered at the max
-              and min corner cases and for the middle values
-            * Data bus has functional coverage using walking ones and zeroes
-            * One bit width variables are wrapped in one cover point
-              while counters and data have designated cover points
-            * Channel enable signal is stand alone cover point
+            Covers the interrupt done and event done for channel 0 and 1.
             '''
     }
     {
       name: roll_cg
       desc: '''
-            Covers that all the counters revert to zero when they reach
-            the maximum value.
-            Individual initial and maximum counter values
+            Covers the individual initial and maximum counter values
```

Comment:
If we only cover the initial and max counter value, how do we know if the counter rolls over to the next round? This coverpoint does not seem to match the name :)

Reply:
This covergroup is not implemented in the current pattgen_cov_if.sv file and it is included in PR12330. It checks the inital and maximum value of the counters. The reset value is checked by the covergroup pattgen_ch0_cnt_reset_cg and  pattgen_ch1_cnt_reset_cg which are already in place with names ch0_cnt_reset_cg and ch1_cnt_reset_cg.